src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const newObject = {};
+
+  sourceString.split(';').forEach((rule) => {
+    let [property, value] = rule.split(':');
+
+    if (property && value) {
```

Comment:
Consider adding a check to ensure that `property` and `value` are not empty strings. This will prevent adding empty keys or values to the `newObject`, which could lead to unexpected behavior.